qiita_pet/handlers/api_proxy/tests/test_studies.py

```diff
@@ -52,28 +52,79 @@ def test_study_get_req(self):
                     'Future studies will attempt to analyze the soils and '
                     'rhizospheres from the same location at different time '
                     'points in the plant lifecycle.',
-                    'status': 'private',
-                    'spatial_series': False,
-                    'study_description': 'Analysis of the Cannabis Plant '
-                                         'Microbiome',
-                    'shared_with': ['shared@foo.bar'],
-                    'lab_person': {'affiliation': 'knight lab',
-                                   'name': 'LabDude',
-                                   'email': 'lab_dude@foo.bar'},
-                    'principal_investigator': {'affiliation': 'Wash U',
-                                               'name': 'PIDude',
-                                               'email': 'PI_dude@foo.bar'},
-                    'study_alias': 'Cannabis Soils',
-                    'study_id': 1,
-                    'most_recent_contact': datetime(2014, 5, 19, 16, 11),
-                    'publications': [['10.100/123456', '123456'],
-                                     ['10.100/7891011', '7891011']],
-                    'num_samples': 27,
-                    'study_title': 'Identification of the Microbiomes for '
-                                   'Cannabis Soils',
-                    'number_samples_collected': 27}}
+                'status': 'private',
+                'spatial_series': False,
+                'study_description': 'Analysis of the Cannabis Plant '
+                                     'Microbiome',
+                'shared_with': ['shared@foo.bar'],
+                'lab_person': {'affiliation': 'knight lab',
+                               'name': 'LabDude',
+                               'email': 'lab_dude@foo.bar'},
+                'principal_investigator': {'affiliation': 'Wash U',
+                                           'name': 'PIDude',
+                                           'email': 'PI_dude@foo.bar'},
+                'study_alias': 'Cannabis Soils',
+                'study_id': 1,
+                'most_recent_contact': datetime(2014, 5, 19, 16, 11),
+                'publications': [['10.100/123456', '123456'],
+                                 ['10.100/7891011', '7891011']],
+                'num_samples': 27,
+                'study_title': 'Identification of the Microbiomes for '
+                               'Cannabis Soils',
+                'number_samples_collected': 27}}
         self.assertEqual(obs, exp)
 
+        # Test with no lab person
```

Comment:
@ackermag Lab person was never required. When you create a study, you are forced to introduce a PI but not a Lab person. Since it is not required there, I can't make sure that it exists here, which breaks the new interface. Making lab person required involves more changes, which I think out of the scope of this PR and the changes performed here - can you open an issue to discuss this in there? You can also bring it up in the next Qiita meeting.

Note that lab person doesn't necessary lists who uploaded the data - that is the owner of the study (or the people who has access to the study). As an example, I can create an study, add Bob as a PI and Alice as lab person, w/o Bob or Alice being involved at all in the study.